packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -205,18 +252,20 @@ export class TableRow<
 
     private dataRecordChanged(): void {
         this.updateCellStates();
+        this.updateCellIndentLevels();
     }
 
     private nestingLevelChanged(): void {
         this.updateCellIndentLevels();
     }
 
+    private isParentRowChanged(): void {
+        this.updateCellIndentLevels();
+    }
```

Comment:
Is this needed? `updateCellIndentLevels` doesn't use `isParentRow`.

Reply:
I think you are right. For some reason I thought I was running into issues with the indentations not updating as expected, but I can't seem to replicate that, so I've removed these changes.